include/aos/common/pkcs11/pkcs11.hpp

```diff
@@ -577,22 +577,41 @@ class LibraryContext : public PKCS11LibraryContext {
     Error GetLibInfo(LibInfo& libInfo) const;
 
     /**
-     * Opens a session between an application and a token in a particular slot.
+     * Opens a session between an application and a token in a particular slot or returns one from the pool.
      *
      * @param slotID slot identifier.
      * @param flags  indicates the type of session.
      * @return RetWithError<SharedPtr<SessionContext>>.
      */
-    RetWithError<SharedPtr<SessionContext>> OpenSession(SlotID slotID, uint32_t flags);
+    RetWithError<SharedPtr<SessionContext>> OpenSession(SlotID slotID, Flags flags);
```

Comment:
Remove extra space in comment `@param flags  indicates the type of session.`

Reply:
fixed